alembic/versions/64c949860b27_initial_migrations.py

```diff
@@ -0,0 +1,48 @@
+"""Initial migrations
+
+Revision ID: 64c949860b27
+Revises: d00070a49ee4
+Create Date: 2023-12-27 16:54:25.526984
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = '64c949860b27'
+down_revision: Union[str, None] = 'd00070a49ee4'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('city',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(length=255), nullable=False),
+    sa.Column('additional_info', sa.String(length=511), nullable=False),
+    sa.PrimaryKeyConstraint('id')
+    )
```

Comment:
you have initial migrations, which downgrade previous migration which doesnt exist, so it won't work, you should delete all migrations and recreate initial, or your startapp wont work at all